### PR TITLE
Added faq block to offer details and made responsive mobile

### PR DESCRIPTION
**Pull request by mxrcury**
Added faq block to offer details and responsive for mobile
Screenshot from 2023-05-14 20-44-52

Screenshot from 2023-05-14 20-44-39

### CODE REVIEW

File: src/components/multi-accordion-with-title/MultiAccordionWIthTitle.tsx [hunk 1]
```diff
@@ -24,6 +25,8 @@ const MultiAccordionWithTitle: FC<MultiAccordionWithTitleProps> = ({
 }) => {
   const { t } = useTranslation()
 
+  const { isMobile } = useBreakpoints()
```
Comment (MickhaelSh) [inline src/components/multi-accordion-with-title/MultiAccordionWIthTitle.tsx:28]: if you use isMobile only for styles. remove it, and use brekpoints in styles, such as xs,sm etc

File: src/components/multi-accordion-with-title/MultiAccordionWithTitle.styles.ts [hunk 2]
```diff
@@ -7,18 +7,23 @@ const borderStyles = {
     borderRadius: '6px'
 }
 
+const accordionTitleStyles = (isMobile: boolean) => ({
```
Comment (MickhaelSh) [inline src/components/multi-accordion-with-title/MultiAccordionWithTitle.styles.ts:10]: I don't see where this is used, is it needed at all?

File: src/components/multi-accordion-with-title/MultiAccordionWIthTitle.tsx (hunk 1 above)
Comment (mxrcury) [reply src/components/multi-accordion-with-title/MultiAccordionWIthTitle.tsx:28]: oh, you are right, thank you)

File: src/components/multi-accordion-with-title/MultiAccordionWithTitle.styles.ts [hunk 3]
```diff
@@ -7,18 +7,23 @@ const borderStyles = {
...
-        mb: '18px',
+        m: '35px 60px 18px 60px',
     },
-    container: { padding: '30px 60px 65px', ...borderStyles, margin: '10px' },
+    container: { ...borderStyles, pb: '65px', backgroundColor: 'white' },
     icon: { color: 'primary.500', fontSize: '13px' },
-    withIcon: {
-        accordion: { ...borderStyles, mb: '16px', '&:last-child': { mb: 0 }, '&::before': { display: 'none' } },
+    withIcon: (isMobile) => ({
+        accordion: { ...borderStyles, mb: '16px', '&:last-child': { mb: 0 }, '&::before': { display: 'none' }, m: isMobile ? '10px 14px' : '0 60px 16px', p: '12px 0' },
```
Comment (MickhaelSh) [inline src/components/multi-accordion-with-title/MultiAccordionWithTitle.styles.ts:23]: fix your prettier config. code should be formated

File: src/types/common/enums/common.enums.ts [hunk 4]
```diff
@@ -35,7 +35,8 @@ export enum VariantEnum {
   Button = 'button',
   H5 = 'h5',
   H6 = 'h6',
-  Body2 = 'body2'
+  Body2 = 'body2',
```
Comment (MickhaelSh) [inline src/types/common/enums/common.enums.ts:38]: if you alredy modify this enum. split it to typographyVariantEnum and buttonVariantEnum

File: src/pages/offer-details/OfferDetails.tsx [hunk 5]
```diff
@@ -74,6 +78,10 @@ const OfferDetails = () => {
           onBookmarkClick={onBookmarkClick}
         />
       </AppCard>
+      <MultiAccordionWithTitle
```
Comment (MickhaelSh) [inline src/pages/offer-details/OfferDetails.tsx:81]: your component should be wrapped in appCard as other component on page, or styles should be passed to it. don`t use styles needed only for one page inside of reusable component

File: src/components/multi-accordion-with-title/MultiAccordionWIthTitle.tsx [hunk 6]
```diff
@@ -39,7 +42,10 @@ const MultiAccordionWithTitle: FC<MultiAccordionWithTitleProps> = ({
   }
 
   const accordionTitle = title && (
-    <Typography sx={sx.title} variant={VariantEnum.H5}>
+    <Typography
+      sx={{ ...sx.title, ...(isMobile && { m: '16px 20px 10px' }) }}
+      variant={isMobile ? VariantEnum.H6 : VariantEnum.H5}
```
Comment (Yurenko) [inline src/components/multi-accordion-with-title/MultiAccordionWIthTitle.tsx:47]: I'm not sure that it needs here. 
...(isMobile && { m: '16px 20px 10px' })

**Review by maarekua — APPROVED** [review]

File: src/components/multi-accordion-with-title/MultiAccordionWithTitle.styles.ts (hunk 3 above)
Comment (mxrcury) [reply src/components/multi-accordion-with-title/MultiAccordionWithTitle.styles.ts]: okey

File: src/components/multi-accordion-with-title/MultiAccordionWithTitle.styles.ts (hunk 2 above)
Comment (mxrcury) [reply src/components/multi-accordion-with-title/MultiAccordionWithTitle.styles.ts]: already removed

File: src/types/common/enums/common.enums.ts (hunk 4 above)
Comment (mxrcury) [reply src/types/common/enums/common.enums.ts:38]: i didn't get what do you mean where to split it

File: src/pages/offer-details/OfferDetails.tsx (hunk 5 above)
Comment (mxrcury) [reply src/pages/offer-details/OfferDetails.tsx:81]: okey, changed

File: src/components/multi-accordion-with-title/MultiAccordionWIthTitle.tsx (hunk 6 above)
Comment (mxrcury) [reply src/components/multi-accordion-with-title/MultiAccordionWIthTitle.tsx]: removed

File: src/components/multi-accordion-with-title/MultiAccordionWIthTitle.tsx [hunk 7]
```diff
@@ -39,7 +42,10 @@ const MultiAccordionWithTitle: FC<MultiAccordionWithTitleProps> = ({
   }
 
   const accordionTitle = title && (
-    <Typography sx={sx.title} variant={VariantEnum.H5}>
+    <Typography
+      sx={sx.title}
+      variant={isMobile ? VariantEnum.H6 : VariantEnum.H5}
```
Comment (Tolik170) [inline src/components/multi-accordion-with-title/MultiAccordionWIthTitle.tsx:47]: also, you can do this in styles and do not export this enum

Comment (mxrcury) [reply src/components/multi-accordion-with-title/MultiAccordionWIthTitle.tsx:47]: yeap, thx

File: src/types/common/enums/common.enums.ts (hunk 4 above)
Comment (Tolik170) [reply src/types/common/enums/common.enums.ts:38]: As I understand you this enum contain variants for buttons and typography, Mickhael say, that would be better to create two enums instead, typographyVariantEnum and buttonVariantEnum

**Review by MickhaelSh — APPROVED** [review]

**Pull request merged**